todoapp/app.py

```diff
@@ -40,13 +40,13 @@ def create_app():
     app.config.from_object('todoapp.flask_config.Config')
 
     file_path = find_dotenv('.env')
-    load_dotenv(file_path, override=True)
+    load_dotenv(file_path, override=False)
 
     CLIENT_ID = os.getenv("CLIENT_ID")
     CLIENT_SECRET = os.getenv("CLIENT_SECRET")
 
     # session
-    app.config['SECRET_KEY'] = ''.join(random.choice(string.ascii_letters) for _ in range(10))
+    app.config['SECRET_KEY'] = 'SFGBSRTHRYHDSFVSGBTYHYWG'
```

Comment:
This would still be better as an environment variable so we could vary it and avoid sharing production secrets with every developer